Tests/ViewInspectorTests/SwiftUI/CustomViewModifierTests.swift

```diff
@@ -177,6 +177,144 @@ final class ModifiedContentTests: XCTestCase {
+            .modifier(AllowHitTestingTransitiveModifier())
+            .modifier(AllowHitTestingTransitiveModifier())
+            .modifier(AllowHitTestingTransitiveModifier())
+            .modifier(AllowHitTestingTransitiveModifier())
+            .modifier(AllowHitTestingTransitiveModifier())
+            .modifier(AllowHitTestingTransitiveModifier())
+            .modifier(AllowHitTestingTransitiveModifier())
+            .modifier(AllowHitTestingTransitiveModifier())
+
+        let sut = try view.inspect()
+        XCTAssertEqual(sut.content.medium.transitiveViewModifiers.count, 15)
+        XCTAssertEqual(sut.content.medium.viewModifiers.count, 15)
+    }
+
+    func testMultipleEnvironmentModifiers() throws {
+        let view = Text("str")
+            .environment(\.font, .headline)
+            .environment(\.accessibilityEnabled, true)
+            .environment(\.accessibilityEnabled, true)
+            .environment(\.accessibilityEnabled, true)
+            .environment(\.accessibilityEnabled, true)
+            .environment(\.accessibilityEnabled, true)
+            .environment(\.accessibilityEnabled, true)
+            .environment(\.accessibilityEnabled, true)
+            .environment(\.accessibilityEnabled, true)
+            .environment(\.accessibilityEnabled, true)
+            .environment(\.accessibilityEnabled, true)
+            .environment(\.accessibilityEnabled, true)
+            .environment(\.accessibilityEnabled, true)
+            .environment(\.accessibilityEnabled, true)
+            .environment(\.accessibilityEnabled, true)
+
+        let sut = try view.inspect()
+        XCTAssertEqual(sut.content.medium.environmentModifiers.count, 15)
+        XCTAssertEqual(sut.content.medium.viewModifiers.count, 0)
+    }
+
+    func testMultipleCustomEnvironmentModifiers() throws {
+        let view = Text("str")
+            .modifier(AccessibilityEnabledEnvironmentModifier())
+            .modifier(AccessibilityEnabledEnvironmentModifier())
+            .modifier(AccessibilityEnabledEnvironmentModifier())
+            .modifier(AccessibilityEnabledEnvironmentModifier())
+            .modifier(AccessibilityEnabledEnvironmentModifier())
+            .modifier(AccessibilityEnabledEnvironmentModifier())
+            .modifier(AccessibilityEnabledEnvironmentModifier())
+            .modifier(AccessibilityEnabledEnvironmentModifier())
+            .modifier(AccessibilityEnabledEnvironmentModifier())
+            .modifier(AccessibilityEnabledEnvironmentModifier())
+            .modifier(AccessibilityEnabledEnvironmentModifier())
+            .modifier(AccessibilityEnabledEnvironmentModifier())
+            .modifier(AccessibilityEnabledEnvironmentModifier())
+            .modifier(AccessibilityEnabledEnvironmentModifier())
+            .modifier(AccessibilityEnabledEnvironmentModifier())
+
+        let sut = try view.inspect()
+        XCTAssertEqual(sut.content.medium.environmentModifiers.count, 15)
```

Comment:
This test fails this assertion in the state of f52f300f10df9a5980dcf839f0048a9dcccf6e72 (the first commit of this branch).

The `environmentModifiers.count` returns **32767** items.

<img width="1101" alt="Screenshot 2023-12-13 at 10 08 42 PM" src="https://github.com/nalexn/ViewInspector/assets/2278022/11807b68-ed13-410d-baa0-5f3489d47db1">